TestSwitchApi/TestSwitchApi/Models/Response/ListResponse.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using TestSwitchApi.Models.Request;
+
+namespace TestSwitchApi.Models.Response
+{
+   public class ListResponse<T>
+    {
+        private readonly string _path;
+        public IEnumerable<T> Items { get; }
+        public int TotalNumberOfItems { get; }
+        public int Page { get; }
+        public int PageSize { get; }
+
+        public string NextPage
+        {
+            get
+            {
+                if (!HasNextPage())
+                {
+                    return null;
+                }
+
+                var url = $"/{_path}?page={Page + 1}&pageNumber={PageSize}";
```

Comment:
sorry - missed this before.

Should this be `pageSize` instead of `pageNumber`?